src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/ResultItem.tsx

```diff
@@ -296,17 +307,35 @@ export const ResultItem: React.FC<ResultItemProps & AccessibilityProps> = ({
             </View>
           </View>
         </View>
-        <ResultItemFooter />
+        <ResultItemFooter bookingRequirement={tripPattern.bookingRequirement} />
       </Animated.View>
     </TouchableOpacity>
   );
 };
-function ResultItemFooter() {
+
+const ResultItemFooter: React.FC<{
+  bookingRequirement: BookingRequirement;
+}> = ({bookingRequirement}) => {
   const styles = useThemeStyles();
   const {t} = useTranslation();
 
+  const {requiresBooking, requiresBookingUrgently} = bookingRequirement;
+
   return (
     <View style={styles.resultFooter}>
+      <View style={styles.footerNotice}>
+        {(requiresBooking || requiresBookingUrgently) && (
```

Comment:
requiresBooking is always true if requiresBookingUrgently is true?
```suggestion
        {requiresBooking && (
```